maintainers/scripts/update-luarocks-packages

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p nix-prefetch-scripts luarocks-nix -i bash
+
+# You'll likely want to use
+# ``
+# nixpkgs $ maintainers/scripts/update-luarocks-packages pkgs/development/lua-modules/generated-packages.nix
+# ``
+# to update all libraries in that folder.
+# to debug, redirect stderr to stdout with 2>&1
+
+
+# stop the script upon C-C
+set -eu -o pipefail
+
+if [ $# -lt 1 ]; then
+    echo "Usage: $0 GENERATED_FILENAME"
+    echo "(most likely pkgs/development/lua-modules/generated-packages.nix)"
+    exit 1
+fi
+
+GENERATED_NIXFILE="$1"
+PACKAGE=""
+# we have to use a temp file else
+TMP_FILE="$(mktemp)"
+
+exit_trap()
+{
+  local lc="$BASH_COMMAND" rc=$?
+  test $rc -eq 0 || echo "*** error $rc: $lc"
```

Comment:
Maybe also report `$TMP_FILE` for inspection/removal?

Aesthetically I would prefer reporting errors with `>&2`.

Reply:
Done